Sequencer/Sequencer/TcSequencer/Example/Mock_EM.TcPOU

```diff
@@ -0,0 +1,86 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1">
+  <POU Name="Mock_EM" Id="{2fbe5d7f-06dd-041a-2d03-5fb4cef27abb}" SpecialFunc="None">
+    <Declaration><![CDATA[
+FUNCTION_BLOCK Mock_EM IMPLEMENTS IMOCK_EM
```

Comment:
This appears to be the only place where _'s are used in the name. Is there significance to IMOCK being all-caps?